plugins/item_tasks/plugin_tests/tasks_test.py

```diff
@@ -103,12 +103,16 @@ def testAddItemTasksToFolderFromJson(self):
         with open(os.path.join(os.path.dirname(__file__), 'spec.json')) as f:
             spec = f.read()
         parsedSpec = json.loads(spec)
+
+        token = self.model('token').createToken(
+            user=self.admin, scope='item_task.set_task_spec.%s' % folder2['_id'])
```

Comment:
I fixed it up in every place I could, but this particular one I left as an artificially created token since we don't run the job creation endpoint in advance of that one.